lib/puppet/type/gluster_volume.rb

```diff
@@ -80,6 +80,19 @@ def property_matches?(current, desired)
     munge { |val| Array(val) }
   end
 
+  newparam(:local_peer_aliases, :array_matching => :all) do
+    desc([
+        'Other names for the current host.',
+        'Anything included here will be ignored when checking peers.',
+      ].join(' '))
```

Comment:
Oh, right. All the multiline syntaxes have indentation issues and I'd need to postprocess the values to make them look sane, so this is probably about the cleanest we'll get. :-/